### PR TITLE
RHCLOUD-31712 Improve integration-disabled emails in case of IOException

### PR DESCRIPTION
**Pull request by gwenneg**

### CODE REVIEW

File: engine/src/main/resources/templates/drawer/Integrations/failedIntegrationDrawerBody.html
Comment (gwenneg) [inline engine/src/main/resources/templates/drawer/Integrations/failedIntegrationDrawerBody.html:1]: This file should have been deleted in an older PR.

File: engine/src/main/java/com/redhat/cloud/notifications/events/EndpointErrorFromConnectorHelper.java [hunk 1]
```diff
@@ -52,45 +63,60 @@ public void manageEndpointDisablingIfNeeded(Endpoint endpoint, JsonObject payloa
...
-                     * server failure, which is usually something temporary. Sending another notification to
-                     * the same endpoint may work in the future, so the endpoint is only disabled if the max
-                     * number of endpoint failures allowed from the configuration is exceeded.
-                     */
-                    boolean disabled = endpointRepository.incrementEndpointServerErrors(endpoint.getId(), maxServerErrors);
-                    if (disabled) {
-                        disabledWebhooksServerErrorCount.increment();
-                        Log.infof("Endpoint %s was disabled because we received too many 5xx status while calling it", endpoint.getId());
-                        integrationDisabledNotifier.tooManyServerErrors(endpoint, maxServerErrors);
+                    if (HTTP_SERVER_ERRORS.contains(httpErrorType.get())) {
```
Comment (g-duval) [inline engine/src/main/java/com/redhat/cloud/notifications/events/EndpointErrorFromConnectorHelper.java:91]: Shouldn't it be an `else if` ?

**Review by g-duval — APPROVED** [review]
One minor comment, LGTM

File: engine/src/main/java/com/redhat/cloud/notifications/events/EndpointErrorFromConnectorHelper.java (hunk 1 above)
Comment (gwenneg) [reply engine/src/main/java/com/redhat/cloud/notifications/events/EndpointErrorFromConnectorHelper.java]: It should indeed. Fixed!

**Pull request merged**